build.gradle

```diff
@@ -127,8 +127,6 @@ dependencies {
     implementation 'org.springframework:spring-webflux'
     implementation 'org.springframework.boot:spring-boot-starter-reactor-netty'
 
-    implementation 'com.nimbusds:nimbus-jose-jwt:8.11' // pinned due to CVE-2019-17195 in 7.8.1 brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
```

Comment:
not needed as pinning spring security to `5.3.0.RELEASE` brings in a fixed version anyway